README.md

```diff
@@ -90,20 +110,16 @@ Hint: `/path/to/KLIFS_download` means `/path/to/folder/with/KLIFS_download/folde
 ##### 1. Preprocessing
 
 ```bash
-kffl-preprocessing 
--f /path/to/KLIFS_download 
--o put/path/to/FragmentLibrary
+kffl-preprocessing -f /path/to/KLIFS_download -o /put/path/to/FragmentLibrary
 ```
 * The output file ```/path/to/KLIFS/data/KLIFS_download/filtered_ligands.csv``` contains metadata on all ligands 
 that were chosen for the fragmentation. 
-* Metadata on the discarded ligands is written to ```put/path/to/FragmentLibrary/discarded_ligands/preprocessing.csv```.
+* Metadata on the discarded ligands is written to ```/put/path/to/FragmentLibrary/discarded_ligands/preprocessing.csv```.
 
 ##### 2. Fragmentation
 
 ```bash
-kffl-fragmentation 
--f /path/to/KLIFS_download 
--o /path/to/fragment_library
+kffl-fragmentation -f /path/to/KLIFS_download -o /path/to/fragment_library
```

Comment:
Is the output path supposed to be the same as in 1.? I would suggest to keep that consistent